### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Baschdl**
Merge #444 first

### CODE REVIEW

**Review by bjrne — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
+# README
 # Technische Dokumentation
```
Comment (bjrne) [inline README.md]: German wording, also does not really make sense anymore

**Review by bjrne — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (feeds) [reply README.md]: I fixed it :)

**Comment by feeds** [comment]
#444 has been merged

**Review by feeds — APPROVED** [review]

**Comment by feeds** [comment]
I opened another issue for making the readme truly prettier. Its just a list of good things to know right now

**Pull request merged**